creator/data_templates/mutations/data_template.py

```diff
@@ -0,0 +1,217 @@
+import graphene
+from graphql import GraphQLError
+from graphql_relay import from_global_id
+from django.conf import settings
+
+from creator.organizations.models import Organization
+from creator.data_templates.models import DataTemplate
+from creator.data_templates.nodes.data_template import DataTemplateNode
+
+
+class CreateDataTemplateInput(graphene.InputObjectType):
+    """Parameters used when creating a new data_template"""
+
+    name = graphene.String(
+        required=True, description="The name of the data_template"
+    )
+    description = graphene.String(
+        required=True, description="The description of the data_template"
+    )
+    icon = graphene.String(
```

Comment:
do we still want to use icons for this? I remember chris saying he could never remember what the icons meant and preferred labels. I could see picking a unique meaningful icon for a template being a bit difficult.

Reply:
I wasn't sure about this..I'm open to changing it to label 🤷‍♀️

Reply:
I left it as icon for now. We could always add a label field later as well